releasenotes/notes/27239.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: security-fix
+area: security
+issue:
+  - 27238
+releaseNotes:
+  - |
+    **Fixed** supports cacert's algorithms other than RSA.
```

Comment:
Not sure exactly what this means, and relating it to the comment in the PR doesn't help.

Maybe:
` **Added** support for cacert algorithms other than RSA.`  Or maybe it's we `**Fixed**` support since it was already supposed to be there?

Reply:
I think it's a "Fix" because the support for other algorithms is supposed to be there. It's failed because of the bug that this pr intends to fix.

Reply:
> Not sure exactly what this means, and relating it to the comment in the PR doesn't help.
> 
> Maybe:
> ` **Added** support for cacert algorithms other than RSA.` Or maybe it's we `**Fixed**` support since it was already supposed to be there?

The istio website does not have any certificate indicating which algorithm is supported, that is, it is understood by default to support common certificates.
So I think it is a `**Fixed**`.

Reply:
Sugguest:
`**Fixed** the CA's certificate signature algorithm to be the default algorithm corresponding to the CA's signing key type.`